common/config/azure-pipelines/ci.yaml

```diff
@@ -26,7 +26,7 @@ schedules:
 jobs:
   - template: jobs/ci-core.yaml
     parameters:
-      name: Node_16
-      nodeVersion: 16.13.0
+      name: Node_18
+      nodeVersion: 18.x
```

Comment:
Can you switch this to 18.12.0 as well so it matches fast-ci.yaml?

Reply:
Both are now 18.12.x